### PR TITLE
Textarea input onchange

### PR DESCRIPTION
**Pull request by austinknight**

### CODE REVIEW

File: src/components/TextareaInput/TextareaInput.js [hunk 1]
```diff
@@ -254,28 +244,43 @@ class TextareaInput extends React.Component {
     }
   }
 
+  onChange = (e) => {
+    if (e) {
+      e.preventDefault();
+    }
+
+    this.setState({
+      value: e.target.value
```
Comment (seanevd) [inline src/components/TextareaInput/TextareaInput.js]: shouldn't this be inside the if (e) statement?

File: src/components/TextareaInput/TextareaInput.js [hunk 2]
```diff
@@ -254,28 +244,43 @@ class TextareaInput extends React.Component {
...
+    }, () => {
+      if (this.props.onChange) {
+        this.props.onChange(this.textareaInput.value);
```
Comment (seanevd) [inline src/components/TextareaInput/TextareaInput.js]: Does this create an infinite loop? Also, does `this.textareaInput` return anything anymore? I thought you removed the ref on `<Textarea>`.

File: src/components/TextareaInput/TextareaInput.js (hunk 1 above)
Comment (austinknight) [reply src/components/TextareaInput/TextareaInput.js]: yes

File: src/components/TextareaInput/TextareaInput.js (hunk 2 above)
Comment (austinknight) [reply src/components/TextareaInput/TextareaInput.js]: good catch

**Pull request merged**